ci/scrape-targets.py

```diff
@@ -0,0 +1,15 @@
+import sys
+from bs4 import BeautifulSoup
+
+html = open(sys.argv[1], 'r').read()
+soup = BeautifulSoup(html, features="html.parser")
+# The tables are:
+# Tier 1
+# Tier 2 with host tools     <-- we want this one
+# Tier 2 without host tools  <-- and also this
+# Tier 3
+for table in soup.find_all("table")[2:4]
```

Comment:
So `2:4` means "indices 1 and 2"? Or is there something else going on here that I don't understand?

Reply:
Nope, I simply typed the wrong numbers